100joon/나는야-포켓몬-마스터-이다솜-1620.js

```diff
@@ -0,0 +1,14 @@
+const fs = require("fs");
+const input = fs.readFileSync("/dev/stdin").toString().split("\n");
+
+const questionCount = input.shift();
+const question = input.splice(Number(questionCount.split(" ")[0]), );
+question.splice(Number(questionCount.split(" ")[1]), );
+const inputObj = {};
+
+input.forEach((el, idx) => inputObj[el] = idx);
+
+for (let el of question) {
+    if (isNaN(el)) console.log(inputObj[el] + 1);
```

Comment:
[Number.isNaN 쓰기](https://github.com/airbnb/javascript#standard-library--isnan)
```suggestion
    if (Number.isNaN(el)) console.log(inputObj[el] + 1);
```